### PR TITLE
Week3 ex 1-3

### PR DESCRIPTION
**Pull request by gmakrakis**

### CODE REVIEW

File: Week1/Week-1/w1ex3.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    xhr.onload = function () {
+        if (xhr.status == 200) {
+            const loadData = JSON.parse(xhr.responseText);
+            console.log(loadData);
+
+            const createLi = document.createElement('li');
+            document.getElementById('list').appendChild(createLi);
+
+            const createImgElement = document.createElement('img');
+            const addAttribute = createImgElement.setAttribute('src', loadData.message);
```
Comment (nikoskleidis) [inline Week1/Week-1/w1ex3.js:18]: setAttribute doesn't return a value. No need for addAttribute variable here

File: Week1/Week-1/w1ex3.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+function loadaxios() {
+    let axiosFetch = () => {
+        axios
+            .get('https://dog.ceo/api/breeds/image/random')
+            .then(function (response) {
+                console.log(response.data);
+                const createLi = document.createElement('li');
+                document.getElementById('list').appendChild(createLi);
+                const createImgElement = document.createElement('img');
```
Comment (nikoskleidis) [inline Week1/Week-1/w1ex3.js:33]: `createImgElement` is not a proper name for a variable. It suits more to a function. In general verbs are used as prefixes to describe functions eg isActive, createElement, calculateAverage. So here I propose a different name like `imgElement`

File: Week1/Week-1/w1ex3.js (hunk 1 above)
Comment (gmakrakis) [reply Week1/Week-1/w1ex3.js:18]: The point was to append the link that I get from the api in img src tag. That's why I used setAttribute. You mean that it doesn't work ? O.o

Comment (nikoskleidis) [reply Week1/Week-1/w1ex3.js:18]: you are correctly using setAttribute here. What I am saying is that setAttribute doesn't return anything. You are assigning the returned value of setAttribute to the addAttribute variable. This assignment will result to setting addAttribute variable to undefined. You later are not using the addAttribute variable anywhere. So this assignment is not needed. It should be just 
`createImgElement.setAttribute('src', loadData.message);`

**Pull request merged**